src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -459,7 +459,11 @@ def convert_epa_params_to_mvs(epa_dict):
             logging.info(
                 f"The assets parameters '{MAP_MVS_EPA[asset_group]}' is not present in the EPA parameters to be parsed into MVS json format"
             )
+            epa_dict.update({asset_group: {}})
+            dict_values.update({asset_group: {}})
 
+    # Check if all necessary input parameters are provided
+    # Does this also identify excess parameters? They occur in storage_assets right now with DISPATCHABILITY.
```

Comment:
Yes, `compare_input_parameters_with_reference()` returns a dict with two keys:

1. `MISSING_PARAMETERS_KEY`
2. `EXTRA_PARAMETERS_KEY`

Reply:
Oh, yeah. Added a loop to logg excess parameters now: 0b7bb1ab5f0e5368c9b2f4be8fa13a22bc0275b1 

Excess parameters in the storage sub-assets are not displayed (but this is something that needs to be changed for MVS in general).